BudgetTracker.BudgetSquirrel.Web/Application/Budgeting/BudgetCreationViewModel.cs

```diff
@@ -0,0 +1,26 @@
+using BudgetTracker.Business.Budgeting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BudgetTracker.BudgetSquirrel.Application.Budgeting
+{
+    public class BudgetCreationViewModel
+    {
+        public string Name { get; set; }
+        public double? PercentAmount { get; set; }
+        public decimal? SetAmount { get; set; }
```

Comment:
This is probably fine for now, but might be a good idea to have a small description of the budget. Maybe the name is enough though.